hazelcast/src/test/java/com/hazelcast/spi/discovery/impl/DiscoveryServicePropertiesUtilTest.java

```diff
@@ -120,6 +121,25 @@ public void unknownProperty() {
         // throw exception
     }
 
+    @Test
+    public void nullOptionalProperty() {
+        Map<String, Comparable> properties = singletonMap(PROPERTY_KEY_1, null);
```

Comment:
Super minor: you can add `// given` before this line to follow the convention of other tests.